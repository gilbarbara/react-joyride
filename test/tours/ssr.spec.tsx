/**
 * @vitest-environment node
 */

import React from 'react';
import { renderToStaticMarkup } from 'react-dom/server';

import Standard from '../__fixtures__/Standard';

describe('Joyride > SSR', () => {
  it('should render without errors', async () => {
    const view = renderToStaticMarkup(<Standard />);

    expect(view).toMatchSnapshot();
  });
});
