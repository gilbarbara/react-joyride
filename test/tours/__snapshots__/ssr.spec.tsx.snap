// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Joyride > SSR should render without errors 1`] = `"<div data-test-id="demo"><main><div class="hero"><div class="container"><div class="hero__content"><h1><span>Create walkthroughs and guided tours for your ReactJS apps.</span></h1><button data-test-id="start" type="button">Let&#x27;s Go!</button></div></div></div><div class="demo__section projects"><div class="container"><h2><span>Projects</span></h2><div class="list"><div>Installation</div><div>Documentation</div><div>Support </div></div></div></div><div class="demo__section mission"><div class="container"><h2><span>Mission</span></h2><button data-test-id="mission-button" type="button">Click me</button></div></div><div class="demo__section about"><div class="container"><h2><span>About</span></h2></div></div></main><footer class="demo__footer"><div class="container"><button type="button">Joyride</button></div></footer></div>"`;
